scripts/update-version.mjs

```diff
@@ -0,0 +1,59 @@
+import { existsSync, readFileSync, writeFileSync } from "fs";
+import fetch from "node-fetch";
+import { join } from "path";
+
+// The suffix we use to publish to npm wip version of the libs
+const SUFFIX = "next";
+
+const nextVersion = async ({ project, currentVersion }) => {
```

Comment:
you need `project` here?

Reply:
It's used to get the versions below. So I guess so.

Reply:
I missed it, thanks.